recipe/meta.yaml

```diff
@@ -43,7 +43,12 @@ requirements:
     - libgomp      # [linux or win]
     - ninja
   host:
-    - cccl !=2.4.0,!=2.5.0  # [cuda_compiler != "None"]
+    ############################################################################################
+    # Pin to CCCL 2.7 to avoid build issues in older or newer CCCLs.                           #
+    #                                                                                          #
+    # xref: https://github.com/conda-forge/xgboost-feedstock/pull/219#issuecomment-2738813028  #
+    ############################################################################################
+    - cccl =2.7             # [cuda_compiler != "None"]
```

Comment:
Have pinned to CCCL 2.7, which is the last version of CCCL where this feedstock built successfully

So far that seems to be addressing the issues seen above: https://github.com/conda-forge/xgboost-feedstock/pull/219#issuecomment-2738813028

Reply:
Filed an issue to track unpinning CCCL: https://github.com/conda-forge/xgboost-feedstock/issues/249